src/pages/assets/AssetsOverviewPage.js

```diff
@@ -46,20 +49,28 @@ const AssetsOverview = ({ slugs }) => {
           <FeaturedWatchlist />
         </div>
         <div className={styles.section}>
-          <MyWatchlist />
+          <MyWatchlist isLoggedIn={isLoggedIn} />
```

Comment:
@devanshu-santiment instead of pass isLoggedIn in this component,
make {isLoggedIn && for full section.}